### ISSUE TITLE
Use git tag for each nuget release

### ISSUE THREAD

**Issue by andrekoehler** [xamarin/urho#260]
I have to build my own version of UrhoSharp that uses the Direct3D11 backend in Windows instead of the OpenGL backend because Anti-Aliasing won't work otherwise. So I want to re-build the same versions that are on NuGet, but the time and date on the NuGet page is very inaccurate (+-30 days):
Version | Last updated
1.7.3-pre | 9 days ago
1.7.0-pre | a month ago
1.5.22 (current version) | a month ago
1.5.20 | 2 months ago
1.5.10 | 3 months ago

Also, I believe NuGet only shows the date when the package was uploaded, not the date of the commit it was built from.

tl;dr Please create git tags whenever building and uploading a package to NuGet so everyone knows what versions are more stable than others.

**Comment by EgorBo** [comment]
Thanks for noticing that, I am configuring CI to do that. 
Also D3D11 backend will be available in nuget (but OpenGL will still be default)
